src/platform/plugins/shared/discover/public/utils/index.ts

```diff
@@ -7,11 +7,4 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-/*
```

Comment:
moved `loadSharingDataHelpers` to `public/index.ts` to avoid loading `getSortForEmbeddable` in page load bundle.